docs/developers/packaging.md

```diff
@@ -121,27 +121,3 @@ For more in-depth explanations, please visit the following pages:
 - [https://blog.ionelmc.ro/2014/05/25/python-packaging](https://blog.ionelmc.ro/2014/05/25/python-packaging)
 - [https://hynek.me/articles/testing-packaging](https://hynek.me/articles/testing-packaging)
 - [https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout)
-
-
-## Publishing and Distribution
```

Comment:
Describing automation steps somewhere else is notoriously endangered to become outdated. If someone is interested in how the publishing works, they should look at the GitHub actions YAML files, because these cannot lie.